packages/widget_toolkit_pin/lib/src/lib_pin_code_with_biometrics/blocs/pin_code_bloc.dart

```diff
@@ -20,17 +20,18 @@ abstract class PinCodeBlocEvents {
 
   ///Triggered when user tap on biometrics button
   void biometricsButtonPressed();
-
-  void checkIfPinIsStored();
 }
 
 /// A contract class containing all states of the PinCodeBloc.
 abstract class PinCodeBlocStates {
-  ///Indicating how much dots to present
+  /// Emits cyrrent PIN length to UI
   Stream<int> get digitsCount;
 
+  /// Emits placeholder PIN length to UI
+  Stream<int> get placeholderDigitsCount;
+
   ///Indicating the biometrics button must be presented
-  ConnectableStream<bool> get showBiometricsButton;
+  Stream<Result<bool>> get showBiometricsButton;
```

Comment:
Was there a reason for changing the state to a regular stream ?

Reply:
Yes, as a `ConnectableStream` some golden tests weren't behaving as expected (`showBiometricsButton` would get stuck in loading state, and I needed to be able to show it in some tests) so I've switched it to a `ReplayStream`